Allow preventScroll

mattmattvanvoorst opened the pull request:
Added option to prevent the document from scrolling to bring newly focused element into view.

https://developer.mozilla.org/en-US/docs/Web/API/HTMLOrForeignElement/focus

jaredLunde commented:
I dig it! My preferred API for this would be `{includeRoot: boolean, preventScroll: boolean}` as the second argument and bumping this package to a new major version. You cool updating your PRs with that in mind?

mattmattvanvoorst commented:
Definately! Will look at it the moment I have time!

mattmattvanvoorst commented:
@jaredLunde I've updated the PR. The second argument now accepts an object with your proposed syntax, but it also still accepts a boolean for legacy fallback reasons. Your accordion for example won't break now with this new version.

I bumped the version to 2.0.0 and updated readme.

jaredLunde commented:
You're awesome. Thank you very much!
